eslint.config.js

```diff
@@ -20,14 +20,13 @@ export default [
       '**/*.d.ts',
       '**/dist/**/*',
       '**/coverage/**/*',
-      'packages/ai-api/src/client/**/*',
-      'packages/foundation-models/src/azure-openai/client/inference/schema/on-your-data-system-assigned-managed-identity-authentication-options.ts',
+      'packages/ai-api/src/client/**/*'
     ]
   },
   {
     files: [
       '**/test-util/**/*.ts',
-      '**/packages/orchestration/src/client/**/*',
+      '**/packages/**/client/**/*',
```

Comment:
[q] Why don't we ignore all clients above like `ai-api` to ignore all rules?

Reply:
Yeah I guess we could do that. I was not sure if we want to completely turn off linting for them